### ISSUE TITLE
Integrating OpenStreetMap in Map Listing widget 

### ISSUE THREAD

**Issue by tejasbusiness** [CrocoBlock/suggestions#2173]
I was watching the video on using the Maps Listing widget where there is an option to add Google Maps API, which is great.

But I would like to request a feature to be added to this where alternatively if we can integrate OpenStreetMap instead of Google map.

A number of themes like ListingPro have this feature already. Adding this will surely help users who can't afford to pay for Google API and/or users from those countries where Google maps can't be used.

Waiting eagerly for the Team CrocoBlock to add this feature.

Thanks in advance!

Stay safe and stay blessed

Tejas G
India

**Comment by gaiuskong** [comment]
This is extremely important for any website that's doing business in China, or has a good portion of the audience located in China, because Google Maps are banned in China, and OpenStreetMap IS WORKING in China.  

A good Elementor addon called "Element Pack" has OpenStreetMap integration and works great. 

Thank you for working on this!!!

**Comment by dbfischer** [comment]
I agree. Many customers don't like Google Maps because of privacy issues.
OpenStreetMap would be the ideal solution.

Kind regards
Dirk

**Comment by devyncocreative** [comment]
+1 OpenStreetMap for privacy concerns with Google Maps

**Comment by Crocoblock** [comment]
was added and released in 3.0.0 JetEngine
